packages/SketchMorph2-Tests.package/MorphicMonetTests.class/instance/testIsTemporaryTextMorphThere.st

```diff
@@ -0,0 +1,9 @@
+UI existence tests
+testIsTemporaryTextMorphThere
+	| collection |
+	self doButtonAction: M2TextTool label.
+	self subject canvas mouseDown: (self subject generateMouseEvent: #mouseDown button: MTFMorphWrapper leftButton on: self subject canvas).
+	collection := self subject canvas submorphsSatisfying: [:morph | morph class = M2TemporaryTextMorph].
```

Comment:
Is `morph class = M2TemporaryTextMorph` ok to use? Or should we use something like `isM2TemporaryTextMorph`?